### DISCUSSION TITLE
alternative models

### DISCUSSION THREAD

**Discussion by scenaristeur** [Mikubill/sd-webui-controlnet#1293] [Q&A]
as suggested here https://www.reddit.com/r/StableDiffusion/comments/11nk5t3/comment/jbnls8x/?utm_source=reddit&utm_medium=web2x&context=3

can we use those models same way ? https://huggingface.co/webui/ControlNet-modules-safetensors/tree/main

**Comment by lllyasviel** [comment]
the model name in this repo mismatch t2i adapter official names (we need to match yaml) and some t2i models may not work
